src/components/Product.js

```diff
@@ -0,0 +1,75 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+import Card from "@mui/material/Card";
+import CardHeader from "@mui/material/CardHeader";
+import CardMedia from "@mui/material/CardMedia";
+import CardContent from "@mui/material/CardContent";
+import CardActions from "@mui/material/CardActions";
+import Collapse from "@mui/material/Collapse";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
```

Comment:
For code beautification lets extract all these from a single `@mui/material` import.
`import { Card, CardHeader, CardMedia, ... } from "@mui/material"`